README.md

```diff
@@ -4,14 +4,14 @@
 
 Wildfire Predictive Services to support decision making in prevention, preparedness, response and recovery.
 
-
 ## Getting Started
 
 ### Dependencies
 
-* [Node.js](https://nodejs.org/en/) - You’ll need to have 10.16.0 or later version on your local development machine. You can use [nvm](https://github.com/nvm-sh/nvm#installation) (macOS/Linux) or [nvm-windows](https://github.com/coreybutler/nvm-windows#node-version-manager-nvm-for-windows) to switch Node versions between different projects.
+- [Node.js](https://nodejs.org/en/) - You’ll need to have 10.16.0 or later version on your local development machine. You can use [nvm](https://github.com/nvm-sh/nvm#installation) (macOS/Linux) or [nvm-windows](https://github.com/coreybutler/nvm-windows#node-version-manager-nvm-for-windows) to switch Node versions between different projects.
```

Comment:
Do we know that 10.16.* is what we'll have running on our build server? @DerekRoberts ?